src/pinns_pde_solve.jl

```diff
@@ -867,12 +867,7 @@ function get_numeric_integral(strategy, _indvars, _depvars, chain, derivative)
                     end
                 end
 
-                for i in 1:size(cord)[2]
-                    ub__ = @Zygote.ignore getindex(ub_, :,  i)
-                    lb__ = @Zygote.ignore getindex(lb_, :,  i)
-                    integration_arr = hcat(integration_arr ,integration_(cord[:, i], lb__, ub__, flat_θ))
-                end
-                return integration_arr
+                return map((cord__,lb__,ub__) -> integration_(cord__,lb__,ub__,flat_θ), eachcol(cord), eachcol(lb_), eachcol(ub_))
```

Comment:
eachcol with views would be good. Does it automatically use views? How does the profile look?

Reply:
How do you see views being used here? 

Another question: I might need to add another line here to convert my `map((cord__,lb__,ub__) -> integration_(cord__,lb__,ub__,flat_θ), eachcol(cord), eachcol(lb_), eachcol(ub_))` which returns an X element Vector into a 1 by X Matrix. This feels silly but the convergence tests break if I don't do this. What is the most efficient way to do this conversion? 

For now I am doing `reshape(integration_arr, :, length(integration_arr)) ` but this doesn't feel right

Reply:
@ChrisRackauckas (forgot to tag)

Reply:
the column iterators, are they slicing per column or viewing?